website/docs/contributing/developer-guide.md

```diff
@@ -4,25 +4,19 @@ title: Developer guide
 
 ## Introduction {#introduction}
 
-This repository contains two main parts. The backend and the frontend of unleash. The backend is a Node.js application that is built using TypeScript. The frontend is a React application that is built using TypeScript. The backend specific code can be found in the `src` lib folder. The frontend specific code can be found in the `frontend` folder.
+This repository consists of two main parts: the backend and frontend of Unleash. The backend is a Node.js application built with TypeScript, while the frontend is a React application also built with TypeScript. You can find code specific to the backend in the `src` lib folder and code specific to the frontend in the `frontend` folder.
```

Comment:
I like this improvement, makes it easier to read.

Reply:
+1